dependencies.props

```diff
@@ -117,12 +113,12 @@
       <ElementName>ProjectNTfsExpectedPrerelease</ElementName>
       <BuildInfoName>ProjectNTfs</BuildInfoName>
     </XmlUpdateStep>
-    <XmlUpdateStep Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfs-TestILC">
+    <XmlUpdateStep Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfsTestILC">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>ProjectNTfsTestILCExpectedPrerelease</ElementName>
-      <BuildInfoName>ProjectNTfs-TestILC</BuildInfoName>
+      <BuildInfoName>ProjectNTfsTestILC</BuildInfoName>
     </XmlUpdateStep>
-    <XmlUpdateStep Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfs-TestILC-PackageVersion">
+    <XmlUpdateStep Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfsTestILC">
```

Comment:
@joperezr For future reference, the itemspec doesn't affect the updater behavior (such as finding the `PackageId`): this upgrader step looks at all BuildInfos. This is why we have `BuildInfoName` metadata on the above updater is there even though it's the same as the spec.

At first having the spec match was just a style convention, but for BfS composed-build auto-upgrade I'm starting to use it to determine which updaters to apply for a composed build.